Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,19 @@
+class ContactsPhonetic < Formula
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/#{version}.tar.gz"
+  version "0.2"
```

Comment:
This line is needless.
Because Homebrew recognizes version number from url.

If you use following url:
https://github.com/Elethom/ContactsUtil-PhoneticName/archive/0.2.tar.gz

Homebrew recognize **0.2** as version number.